python_examples/age_stratified_example/age_stratified_plot.py

```diff
@@ -221,7 +221,7 @@ def barchart(self, outfile: str,
         if param_file:
             title = 'New ' + title
         plt.title(title)
-        plt.gca().legend().set_title('')
+        plt.gca().legend().set_title('Age Group')
         plt.xlabel("Time")
```

Comment:
Not important but just realized Time label does not seem to be appearing. Do we need a tight_layout.